osu.Game.Tests/Visual/Playlists/TestScenePlaylistsResultsScreen.cs

```diff
@@ -63,20 +63,29 @@ public override void SetUpSteps()
                 userScore.Statistics = new Dictionary<HitResult, int>();
                 userScore.MaximumStatistics = new Dictionary<HitResult, int>();
 
-                bindHandler();
-
                 // Beatmap is required to be an actual beatmap so the scores can get their scores correctly
                 // calculated for standardised scoring, else the tests that rely on ordering will fall over.
                 Beatmap.Value = CreateWorkingBeatmap(Ruleset.Value);
             });
         }
 
+        public void SetUpRequestHandler(bool noScores = false)
+        {
+            AddStep("set up request handler", () =>
+            {
+                bindHandler(noScores: noScores);
+            });
+        }
+
         [Test]
         public void TestShowWithUserScore()
         {
+            SetUpRequestHandler();
+
             AddStep("bind user score info handler", () => bindHandler(userScore: userScore));
```

Comment:
Now clearly `bindHandler` is done twice, which was the case previously also. Tests seem to pass with `SetUpRequestHandler` removed. Should it be removed?

Reply:
Yes.